events/event_queue.go

```diff
@@ -255,6 +262,11 @@ type AsyncEventConsumer struct {
 }
 
 func (t *AsyncEventConsumer) Handle(ctx api.Context) (int, error) {
+	tracer := otel.GetTracerProvider().Tracer("event-tracer")
+	traceCtx, span := tracer.Start(ctx, "event-queue-"+strings.Join(t.watchEvents, "/"))
```

Comment:
I think the watchEvents should be attributes

Reply:
👍🏽